src/test/java/fr/insee/pogues/transforms/visualize/PoguesJSONToPoguesJSONDerefImplTest.java

```diff
@@ -83,7 +84,7 @@ void dereference_nullReferences() throws Exception {
 
         //
         PoguesJSONToPoguesJSONDerefImpl deref = new PoguesJSONToPoguesJSONDerefImpl(questionnairesService);
-        assertThrows(NullReferenceException.class, () -> deref.transformAsQuestionnaire(testedInput));
+        assertThrows(PoguesDeserializationException.class, () -> deref.transformAsQuestionnaire(testedInput));
```

Comment:
The previous error type seems to indicate more clearly what happens in this case: "deserialization exception" doesn't help to understand that the problem is a null reference

Reply:
Error with the mock, i fix it